src/core/grid_based_algorithms/lb_particle_coupling.cpp

```diff
@@ -149,6 +149,9 @@ Utils::Vector3d lb_viscous_coupling(Particle *p,
 namespace {
 bool in_local_domain(Utils::Vector3d const &pos) {
   auto const lblattice = lb_lbfluid_get_lattice();
+  auto const my_left = local_geo.my_left();
```

Comment:
i think this whole function should go to the lattice class, I saw a very similar function before